aop/src/main/java/com/interface21/beans/factory/config/BeanFactoryBeanPostProcessor.java

```diff
@@ -0,0 +1,5 @@
+package com.interface21.beans.factory.config;
+
+public abstract class BeanFactoryBeanPostProcessor implements BeanPostProcessor {
+    public abstract void injectBeanFactory(final Object beanFactory);
+}
```

Comment:
```suggestion
public interface BeanFactoryBeanPostProcessor extends BeanPostProcessor {
    void injectBeanFactory(final Object beanFactory);
}
```

interface 가 아니라 굳이 abstract class 로 하신 이유가 있으시나요?

Reply:
앗 정신 없이 작업 하다 보니 추상클래스로 만들어버렸어요ㅠㅠ
인터페이스로 수정하는 것이 더 좋을 것 같습니다.